Core/clim-basic/windowing/events.lisp

```diff
@@ -102,17 +103,40 @@
          (defmethod event-type ((event ,name))
            ',type)))))
 
+;;; We have three pairs of coordinates. sheet-x and sheet-y are the sheet's
```

Comment:
Maybe
```common-lisp
"
* SHEET-X and SHEET-Y are the position of the pointer event in the coordinate system of the sheet.
* X and Y are the position of the pointer event in the coordinate system of the mirror of the sheet.
* GRAFT-X and GRAFT-Y are the position of the pointer event in the screen coordinate system.
"
```
Could also write one comment above each pair of slots.

Reply:
I'm going to make it
```
;;; We have three pairs of the pointer event coordinates in different
;;; coordinate systems:
;;;
;;; * (X Y) - native coordinates (the mirror)
;;; * (SHEET-X SHEET-Y) - sheet coordinates
;;; * (GRAFT-X GRAFT-Y) - graft coordinates (the screen)
```

"sheet coordinates" and "native coordinates" are terms which may be found in the spec.